src/components/riveMonster/index.ts

```diff
@@ -0,0 +1,51 @@
+import { Rive, Layout, Fit, Alignment } from '@rive-app/canvas';
+
+interface RiveMonsterComponentProps {
+    src: string;  // Path to the .riv file
+    canvas: HTMLCanvasElement;  // Canvas element where the animation will render
+    autoplay: boolean;
+    stateMachines?: string;  // Optional, state machine for animation control
+    fit?: string;  // Fit property (e.g contain, cover, etc.)
+    alignment?: string;  // Alignment property (e.g topCenter, bottomLeft, etc.)
+    width?: number;  // Optional width for the Rive animation
+    height?: number;  // Optional height for the Rive animation
+    onLoad?: () => void;  // Callback once Rive animation is loaded
+}
+
+export class RiveMonsterComponent {
+    private props: RiveMonsterComponentProps;
+    private riveInstance: any;
+  
```

Comment:
Would be great to add a static readonly prop for all the possible monster animation

- Opening Mouth Eat
- Eat Happy
- Idle
- Eat Disgust

and refer to them in `monster.ts` when playing them